runtime/parser/parse_connector.go

```diff
@@ -51,11 +51,17 @@ func (p *Parser) parseConnector(node *Node) error {
 		}
 	}
 
-	// Find out if any properties are templated
```

Comment:
Why remove this comment? If the behavior changed, add a different section comment